src/components/PivotTable/Aggregators/Aggregators.tsx

```diff
@@ -0,0 +1,86 @@
+/** @jsx jsx */
+import { css, jsx } from '@emotion/core'
+
+import { Radio } from '../../Radio'
+import { Select } from '../../Select'
+import { VFlow } from '../../VFlow'
+import { useLocale } from '../../../i18n'
+import { KeyMap } from '../model/model-keyMap'
+import { Aggregator } from './model'
+import { getAggregators, getKeyNotDependentAggregators } from './utils'
+
+interface AggregatorsProps<T extends object> {
```

Comment:
Why is it necessary to extend object?

Reply:
In PEC, the original implementation used `extends any`, which allowed both primitive and non-primitive types. However, extending object was suggested as a way to restrict the type to only non-primitive values